VoiceTest/package.json

```diff
@@ -9,7 +9,7 @@
 	"dependencies": {
 		"react": "16.0.0-beta.5",
 		"react-native": "0.49.2",
-    "react-native-voice": "0.2.0"
+		"react-native-voice": "file:.."
```

Comment:
This might not belong in this PR, but honestly, this is a good idea... In reality, the `VoiceTest` should have the file location from local changes. But for observers and others using the `VoiceTest` with their local changes, I think keeping the npm version on the `master` branch is the way to go.

Reply:
my bad, will deploy a fix soon